webapp/src/libs/ui/ui-button-helm/src/lib/hlm-button.directive.ts

```diff
@@ -13,7 +13,8 @@ export const buttonVariants = cva(
         outline: 'border border-input hover:bg-accent hover:text-accent-foreground',
         secondary: 'bg-secondary text-secondary-foreground hover:bg-secondary/80',
         ghost: 'hover:bg-accent hover:text-accent-foreground',
-        link: 'underline-offset-4 hover:underline text-primary'
+        link: 'underline-offset-4 hover:underline text-primary',
+        navButton: 'font-normal hover:bg-accent hover:text-accent-foreground justify-start gap-2 w-full'
```

Comment:
This is too specific. Just use `ghost` and add your custom classes for the navButton